website/blog/2024-12-16-dso-toolkit-67.2.0.mdx

```diff
@@ -0,0 +1,32 @@
+---
+authors: [hansgrimm]
+---
+
+# DSO Toolkit v67.2.0 👹
+
+Deze release bevat breaking changes voor het volgende component:
```

Comment:
Als deze change niet meer breaking is, waarom staat hier dan nog dat ie breaking is 😛

Reply:
Haha, on it 😉

Reply:
done